src/start.ts

```diff
@@ -54,265 +54,331 @@ export const handler = async (_event: any = {}): Promise<any> => {
           provider
         );
 
+        const airnodeHDNode = ethers.utils.HDNode.fromMnemonic(
+          nodeSettings.airnodeWalletMnemonic
+        );
+
+        const currentBlock = await provider.getBlockNumber();
```

Comment:
I guess I didn't put it in try/catch block just because I probably copied over from [initialize-provider](https://github.com/api3dao/airnode/blob/6d902da259ec3084c8f4764cadc74e270e5c7162/packages/airnode-node/src/evm/handlers/initialize-provider.ts#L56) in node package but you bring up a good point and since this is the first interaction with the provider if it is down or not reachable then the whole process fails which makes me think that I should put it inside a try/catch.